BowlingBall.Tests/GameFixture.cs

```diff
@@ -1,14 +1,88 @@
 using System;
 using Xunit;
-
 namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        Game game=null;
+
+        public GameFixture()
+        {
+            game = new Game();
+        }
+
+        [Fact]
+        public void AllGutterBalls()
+        {
+            RollFrames(0, 0, 10);
+            Assert.Equal(0, game.GetScore());
+        }
+
+        [Fact]
+        public void All2sShouldScore40()
+        {
+            RollFrames(2, 2,10);
+            Assert.Equal(40, game.GetScore());
+        }
+
+        [Fact]
+        public void FirstFrameSpareRest2s()
+        {
+            game.RollSpare(2, 8);
+            RollFrames(2, 2, 9);
+            Assert.Equal(48, game.GetScore());
+        }
+
         [Fact]
-        public void DummyTest()
-        {
-            // This is a dummy test that will always pass.
+        public void FirstTwoFramesSpareRest2sShouldScore56()
+        {
+            game.RollSpare(2, 8);
+            game.RollSpare(2, 8);
+            RollFrames(2, 2, 8);
+            Assert.Equal(56, game.GetScore());
+        }
+
+        [Fact]
+        public void TryingToBowlMoreThan10FramesThrowsOutOfFramesException()
+        {
+            Exception ex = Assert.Throws<IndexOutOfRangeException>(() => RollFrames(2, 2, 11));
+        }
+
+        [Fact]
+        public void FirstFrameStrikeRest2sShouldScore50()
+        {
+            game.RollStrike();
+            RollFrames(2, 2, 9);
+            Assert.Equal(50, game.GetScore());
+        }
+
+        [Fact]
+        public void FirstTwoFramesStrikesRest2sShouldScore68()
+        {
+            game.RollStrike();
+            game.RollStrike();
+            RollFrames(2, 2, 8);
+            Assert.Equal(68, game.GetScore());
+        }
+
+
+        [Fact]
+        public void PerfectGameShouldTotal300()
```

Comment:
Test some more conditions like : 
1. TestAllOnes 
2. TestOneSpare
3. TestOneStrike
4. TestRandomGameNoExtraRoll
5. TestRandomGameWithSpareThenStrikeAtEnd
6. TestRandomGameWithThreeStrikesAtEnd

Reply:
case 1,2,3,6 already covered rest will be Added soon